prod_stack_tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+stacks="api.highfidelity.com \
+	api-pro.highfidelity.com \
+	api-pro-east.highfidelity.com \
+	api-pro-eu.highfidelity.com"
```

Comment:
Hello! Request to pull these stack names out and accept the stack name via command line. Thanks!